### PR TITLE
vdk-jupyter: add UI vdk cell marks

### PR DESCRIPTION
**Pull request by duyguHsnHsn**
What:
Added new design for the cells that are tagged with vdk
The new design includes different background colour of the vdk cells (light blue and dark blue), vdk logo and vdk cell numbering.
Here are the designs in light and dark mode: 
Screenshot 2023-04-11 at 13 59 04
Screenshot 2023-04-13 at 10 58 56

Why: Up until now, we marked the cells for vdk with Jupyter tags, but they are not really user friendly - it is really easy to forget to mark something or mark something by mistake and never realise that you did it since it is really hard to track tags. With these changes it becomes really easy to see which cells are tagged and which are not which makes the UI more user-friendly and error-proof.

Tests: manual tests with dark and light mode 

Signed-off-by: Duygu Hasan [hduygu@vmware.com](mailto:hduygu@vmware.com)

### CODE REVIEW

**Comment by murphp15** [comment]
Are you going to address the codacity warnings?

**Comment by duyguHsnHsn** [comment]
> Are you going to address the codacity warnings?

yes, I am going to do that now, I was looking at something else while all the checks were running

File: projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/vdkTags.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { INotebookTracker } from '@jupyterlab/notebook';
+import { IThemeManager } from '@jupyterlab/apputils';
+
+const addNumberElement = (number: Number, node: Element) => {
+  const numberElement = document.createElement('div');
+  numberElement.innerText = String(number);
+  numberElement.classList.add('jp-vdk-cell-num');
```
Comment (ivakoleva) [inline projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/vdkTags.ts:12]: Why does the numbering start from 0? 😊

**Review by ivakoleva — APPROVED** [review]

File: projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/vdkTags.ts (hunk 1 above)
Comment (duyguHsnHsn) [reply projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/vdkTags.ts:12]: @ivakoleva do you think I should change it to start from 1?

**Review by murphp15 — APPROVED** [review]

File: projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/vdkTags.ts (hunk 1 above)
Comment (ivakoleva) [reply projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/vdkTags.ts:12]: I think we may match ones line numbering of an IDE editor, starting from 1

Comment (duyguHsnHsn) [reply projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/vdkTags.ts:12]: changed

**Pull request merged**